src/components/SwapUkraine.tsx

```diff
@@ -257,14 +257,10 @@ const Swap = () => {
   }, [chainsTokensTools.bridges, chainsTokensTools.exchanges])
 
   useEffect(() => {
-    if (
-      chainsTokensTools.chainsLoaded &&
-      chainsTokensTools.tokensLoaded &&
-      chainsTokensTools.toolsLoaded
-    ) {
+    if (availableChains.length !== 0 && Object.keys(tokens).length !== 0) {
```

Comment:
This check is repeated a lot, maybe extract it to an helper fn?

Reply:
can do!